classes/class-wc-connect-shipping-label.php

```diff
@@ -41,51 +41,60 @@ public function __construct(
 			$this->payment_methods_store = $payment_methods_store;
 		}
 
+		private function get_item_data( WC_Order $order, WC_Order_Item $item ) {
```

Comment:
Let's avoid adding `private` methods going forward unless we need them. It's much easier to test `public` methods.